addons/common/functions/fnc_readSettingFromModule.sqf

```diff
@@ -24,4 +24,4 @@ if (isNil {_logic getVariable _moduleVariable}) exitWith {
 };
 
 // Set the setting globally and force it
-[_settingName, _logic getVariable _moduleVariable, true, true] call FUNC(setSetting);
+[_settingName, _logic getVariable _moduleVariable, true, true] call FUNC(setSetting);
```

Comment:
Apparently your editor is configured to delete newline characters at the end of the files you edit (hence the red symbol in this line), which is the exact opposite to the code standard we have in ACE. Ideally we'd like to have a newline at the end of all files.

Obviously it's not serious though.